src/components/Campaign/CampaignHeader/CampaignHeader.style.ts

```diff
@@ -28,12 +34,8 @@ export const CampaignHeaderBoxBackground = styled(Box)(({ theme }) => ({
   paddingRight: theme.spacing(3),
   paddingTop: theme.spacing(1),
   paddingBottom: theme.spacing(1),
-  [theme.breakpoints.down('md' as Breakpoint)]: {
-    justifyContent: 'center',
-  },
-  [theme.breakpoints.up('md' as Breakpoint)]: {
-    justifyContent: 'flex-end',
-  },
+  backgroundColor: alpha(theme.palette.primary.main, 0.24),
```

Comment:
please avoid using alpha function, `alphaDark500` and `alphaLight500` are doing the same